forge/src/main/java/org/openjdk/skara/forge/HostedRepository.java

```diff
@@ -108,7 +108,7 @@ PullRequest createPullRequest(HostedRepository target,
      * @param authorName  Name of author and committer for commit
      * @param authorEmail Email of author and committer for commit
      */
-    void writeFileContents(String filename, String content, Branch branch, String message, String authorName, String authorEmail);
+    void writeFileContents(String filename, String content, Branch branch, String message, String authorName, String authorEmail, boolean createNewFile);
```

Comment:
Please update the javadoc with the new parameter and make it clear what should happen when set to true or false. If set to true, the call will fail if the file exists. If set to false, the call will fail if the file does not exist.

Reply:
Will do